app/src/main/java/com/hedvig/app/feature/offer/ui/changestartdate/OfferQueryInceptionExt.kt

```diff
@@ -1,12 +1,17 @@
 package com.hedvig.app.feature.offer.ui.changestartdate
 
 import com.hedvig.android.owldroid.fragment.QuoteBundleFragment
+import com.hedvig.app.feature.embark.quotecart.CreateQuoteCartUseCase
 
-fun QuoteBundleFragment.Inception1.toChangeDateBottomSheetData() = ChangeDateBottomSheetData(
+fun QuoteBundleFragment.Inception1.toChangeDateBottomSheetData(
+    quoteCartId: CreateQuoteCartUseCase.QuoteCartId?,
+    quoteNames: List<String>
+) = ChangeDateBottomSheetData(
+    quoteCartId = quoteCartId,
     inceptions = asConcurrentInception?.let { concurrentInception ->
-        concurrentInception.correspondingQuoteIds.map { quoteId ->
+        concurrentInception.correspondingQuoteIds.mapIndexed { index, quoteId ->
             ChangeDateBottomSheetData.Inception(
-                title = "",
+                title = quoteNames.getOrElse(index) { "Insurance #$index" },
```

Comment:
Hmm interesting that we were not showing anything here before. Is this part of what a real user sees? Do we want to localize this to other languages? The word "Insurance" that is.

Reply:
Yeah this is a bit ugly. We should always have the insurance names here, but instead of showing an error I hard coded this text.